contact.html

```diff
@@ -54,38 +54,33 @@
 </ul>
 
 
-<form name="sentMessage" id="contactForm" novalidate>
+<form action="https://docs.google.com/forms/u/0/d/e/1FAIpQLSfX2eMW8InjQMwXqHl8Uk1Xk8-GlwJ275CarAFL-0itDGtBwg/formResponse" method="POST" id="mG61Hd">
```

Comment:
wait we can post directly to a google form from this html form? omg thats great.